docs/core/versions/binding.md

```diff
@@ -0,0 +1,170 @@
+
+Developers should remember the following rules:
+
+- Runtimes are searched for in the following order (the first found is loaded):
+  - Highest patch version (check that the minimum runtime path version is satisfied, otherwise error)
+  - Highest minor version
+- If an acceptable runtime is not found, the application does not run and an error message explains why.
+
+## Publish a self-contained application
+
+You can publish an application as a **self-contained distribution**. This approach includes .NET Core. Self-contained distributions do not have a dependency on runtime environments. Runtime binding occurs at publishing time, not run-time.
+
+The publishing process will select the latest patch version of the given runtime family. For example, `dotnet publish` will select .NET Core 2.0.4 if it is the latest patch version in the .NET Core 2.0 runtime family, independent of the existence of earlier patch versions or minimum runtime patch version values. The publishing process differs from `dotnet build` runtime selection for two reasons:
+
+- The application doesn't specify a hosting environment.
+- The developer generates a final configuration for the application (that can't be practically modified). The best runtime choice is the latest installed runtime patch version. It's the same version used for development and it has the latest (per the machine) security and reliability fixes.
+
+The `runtimeframeworkversion` element will override the default version policy if that element is present in the project file.
+
+It is an error if the minimum version specified for an application is not satisfied. `dotnet publish` binds to latest runtime patch version (within a given major.minor version family). `dotnet publish` does not support the roll-forward semantics of `dotnet run`.
+
+## Publish an application with a specific .NET Core version
+
+`dotnet publish` accepts a version to use for publishing for both self-contained and framework-dependent apps. This is primarily for CI/CD scenarios or other times when you must publish an application, but can't update source code.
+
+You publish with the latest runtime patch version for the given target framework specifying `--latest`. This argument has the same default runtime binding behavior for publishing self-contained applications. See the following example:
+
+``` console
+dotnet publish -c release -o app --latest
+```
+
+You publish with a specific runtime patch version for the given target framework specifying `--version=[version]`. It is an error if this version is missing. It's an error if the given version doesn't support the target framework (as a lower bound). You may specify higher versions (even major versions), provided they exist on the machine. You can see this usage in the following example:
+
+``` console
+dotnet publish -c release -o app --version=3.1.0
```

Comment:
Please tell me that syntax is incorrect. Space not equal is the normal syntax.

Reply:
We should probably use a version that we have even contemplated

Reply:
@KathleenDollard I took that paragraph from the design spec: dotnet/design2#3  It looks like this entire section (Publish an application with a specific .NET Core version) isn't implemented at all. I'm dropping this entire paragraph. Let me know if that's incorrect.